frontend/src/pages/ActivityReport/Pages/components/Objective.js

```diff
@@ -225,15 +224,9 @@ export default function Objective({
     rules: { required: true },
     defaultValue: objective.closeSuspendContext || '',
   });
+  const isOnApprovedReport = objective.onApprovedAR;
```

Comment:
Not sure if we need a useEffect on theses.

Reply:
you do not, no